### PR TITLE
Stargaze studio

### PR DESCRIPTION
**Pull request by merdoyovski**

### CODE REVIEW

File: guides/stargaze-studio/interact-with-a-collection/execute-messages.md [hunk 1]
```diff
@@ -4,17 +4,43 @@
 
 A set of actions can be executed by the admin of the collection. Executable actions are as follows:
 
+* **Mint**
+
+Mint a token from the collection to the caller's address.
+
+* **Purge**
+
+Purge the data stored inside the collection contract. This action is irreversible and requires the collection to be sold out.
```
Comment (MightOfOaks) [inline guides/stargaze-studio/interact-with-a-collection/execute-messages.md:13]:
```suggestion
Purge the minter address data stored inside the minter contract. This action is irreversible and requires the collection to be sold out.
```

File: guides/stargaze-studio/interact-with-a-collection/execute-messages.md [hunk 2]
```diff
@@ -42,33 +80,27 @@ Takes an address and a token ID, then transfers that token to the address.
...
 * **Shuffle the token IDs**
 
-Shuffles all of the token IDs inside the collection.&#x20;
+Shuffles all of the token IDs inside the collection. Collections are created in the same order as they are sent by the user, by default. Executing this message will shuffle all the IDs to make the orders random.
+
+{% hint style="info" %}
+The shuffling price is 500 $STARS.
+{% endhint %}
 
 * **Airdrop Tokens**
```
Comment (MightOfOaks) [inline guides/stargaze-studio/interact-with-a-collection/execute-messages.md:105]: Airdrop Tokens now accepts a .csv file, which includes the header "address,amount", to facilitate airdropping multiple tokens to individual addresses.

File: guides/stargaze-studio/interact-with-a-collection/execute-messages.md [hunk 3]
```diff
@@ -42,41 +80,39 @@ Takes an address and a token ID, then transfers that token to the address.
...
 
-Takes a TXT file that contains Stargaze addresses and airdrops a token for each address.
+Takes a CSV file that contains Stargaze addresses and airdrops a token for each address.
```
Comment (MightOfOaks) [inline guides/stargaze-studio/interact-with-a-collection/execute-messages.md:107]:
```suggestion
Takes a CSV file that contains Stargaze addresses and airdrops the specified number of tokens for each address.
```

File: guides/stargaze-studio/interact-with-a-collection/execute-messages.md [hunk 4]
```diff
@@ -42,41 +80,39 @@ Takes an address and a token ID, then transfers that token to the address.
...
 
-The expected file is a simple .txt file where each line consists of one Stargaze address. The file should look like the following:
+The expected file is a simple .csv file where each line consists of one Stargaze address then a comma followed by the number of tokens to airdrop. The file should look like the following:
 
 ```
-stars1arkrg6nydm9k6d2hwzn9rdqtjtfes58hfqhena
-stars1d90w4s4pcup6qceyrvckj35zwwy2j4u2rqa43t
+stars1arkrg6nydm9k6d2hwzn9rdqtjtfes58hfqhena, 3
```
Comment (MightOfOaks) [inline guides/stargaze-studio/interact-with-a-collection/execute-messages.md:112]:
```suggestion
address,amount
stars1arkrg6nydm9k6d2hwzn9rdqtjtfes58hfqhena,3
```

File: guides/stargaze-studio/interact-with-a-collection/execute-messages.md [hunk 5]
```diff
@@ -42,41 +80,39 @@ Takes an address and a token ID, then transfers that token to the address.
...
+stars1d90w4s4pcup6qceyrvckj35zwwy2j4u2rqa43t, 1
```
Comment (MightOfOaks) [inline guides/stargaze-studio/interact-with-a-collection/execute-messages.md:113]:
```suggestion
stars1d90w4s4pcup6qceyrvckj35zwwy2j4u2rqa43t,1
```

**Comment by merdoyovski** [comment]
Accidentally closed. Reopening.

**Pull request merged**